index.php

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>title</title>
+</head>
+<body>
+<?php
+//This connects to my database
+$db = new PDO('mysql:host=db;dbname=netflix_series','root','password');
+//Fixes the format(one instead of two values)
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,PDO::FETCH_ASSOC);
+// Run $db's query method with the SQL passed to it - from is the table name
+$query = $db-> query('SELECT `id`,`TITLE`,`START DATE`,`END DATE`,`NUMBER OF SEASONS`,`NUMBER OF EPISODES`,`DIRECTOR` FROM `favourites`');
```

Comment:
I would tend to avoid all cap in the db field names

Spaces in field names could potentially cause issues, similar to file names, I would stick with lowercase and underscores, for example

https://anandarajpandey.com/2015/05/10/mysql-naming-coding-conventions-tips-on-mysql-database/